projects/showcase/cypress/support/e2e-driver.ts

```diff
@@ -1,27 +1,28 @@
-import { PNG } from 'pngjs';
-import { MapLibreAssertable, MaplibreCypressHelper } from './maplibre-helper';
+import { PNG } from "pngjs";
```

Comment:
Can this quote change be done using lint? I'm not sure I understand why this is possible - it should be defined in a lint rule and follow it, it shouldn't change as part of a PR without breaking the CI... :-/